model/fv_regional_bc.F90

```diff
@@ -1723,7 +1723,7 @@ subroutine regional_bc_data(Atm,bc_hour                           &
 !***  Sensible temperature
 !--------------------------
 !
-      if (data_source == 'FV3GFS GAUSSIAN NEMSIO FILE') then
+      if ( data_source_group==1 ) then  ! GFS NEMSIO/NETCDF/GRIB2
```

Comment:
Instead of using these magic numbers ( `==1` or `/=1` ) consider using parameters, where parameter with the value of 1 is named  for example `data_source_fv3gfs`. Then you do not need trailing comment to explain the meaning of a comparison. Code becomes self documenting:
`          if ( data_source_group ==  data_source_fv3gfs ) then`
or:
`          if ( data_source_group /=  data_source_fv3gfs ) then`

Reply:
@DusanJovic-NOAA, Completed.